src/routes/components/Table.tsx

```diff
@@ -0,0 +1,401 @@
+            <a href="#sortable-columns">Sortable columns</a>
+            <a href="#number-column">Number columns</a>
+          </GoABlock>
+
+          <h3 id="sortable-columns">Sortable columns</h3>
+          {/*Sortable columns demo - sandbox cannot display users.map and its logic*/}
+          <GoAContainer>
+            <GoATable onSort={sortData}>
+              <thead>
+                <tr>
+                  <th>
+                    <GoATableSortHeader name="firstName">First name</GoATableSortHeader>
+                  </th>
+                  <th>
+                    <GoATableSortHeader name="lastName">Last name</GoATableSortHeader>
+                  </th>
+                  <th>
+                    <GoATableSortHeader name="age" direction="asc">
+                      Age
+                    </GoATableSortHeader>
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                {users.map(user => (
+                  <tr key={user.firstName}>
+                    <td>{user.firstName}</td>
+                    <td>{user.lastName}</td>
+                    <td>{user.age}</td>
+                  </tr>
+                ))}
+              </tbody>
+            </GoATable>
+          </GoAContainer>
+
+          {language === "react" && (
+            <CodeSnippet
+              lang="typescript"
+              tags="react"
+              allowCopy={true}
+              code={`
+             interface User {
+              firstName: string;
+              lastName: string;
+              age: number;
+              }
+              const [users, setUsers] = useState<User[]>([]);
+              useEffect(() => {
+                const _users: User[] = [
+                  {
+                    firstName: "Christian",
+                    lastName: "Batz",
+                    age: 18
+                  },
+                  {
+                    firstName: "Brain",
+                    lastName: "Wisozk",
+                    age: 19
+                  },
+                  {
+                    firstName: "Neha",
+                    lastName: "Jones",
+                    age: 23
+                  },
+                  {
+                    firstName: "Tristin",
+                    lastName: "Buckridge",
+                    age: 31
+                  }
+                ];
+                setUsers(_users);
+              }, []);
+              function sortData(sortBy: string, sortDir: number) {
+                const _users = [...users];
+                _users.sort((a: any, b: any) => {
+                  return (a[sortBy] > b[sortBy] ? 1 : -1) * sortDir;
+                });
+                setUsers(_users);
+              }
+              return (
+                <GoATable onSort={sortData}>
+                  <thead>
+                    <tr>
+                      <th>
+                        <GoATableSortHeader name="firstName">First name</GoATableSortHeader>
```

Comment:
The code snippets allow for whitespace (storybook didn't). So we should keep things readable by adding whitespace where necessary.